invisible_cities/reco/paolina_functions.py

```diff
@@ -69,6 +69,14 @@ def neighbours(va : Voxel, vb : Voxel, scale : float = 1.5) ->bool:
 
     return tuple(nx.connected_component_subgraphs(voxel_graph))
 
+
+def voxels_from_track_graph(track: Graph) ->List[Voxel]:
+    """Create and return a list of voxels from a track graph"""
+
+    voxels = [Voxel(t) for t in t.nodes]
```

Comment:
Firstly, this is just an extremely convoluted and inefficient way of writing `return t.nodes()`. (Actually, there's also a bug in it, the `()` are missing! I think that it does get fixed in a later commit.)

Secondly, I question the value of creating `voxels_from_track_graph(t)` as a synonym for `t.nodes()`. There is some point to it: it creates a name which has a higher level meaning in our context. But I'm far from convinced that it is worth the effort. Given that we use it exactly one place, my temptation is to conclude that we should remove it and just use `track.nodes()` instead.

Reply:
Correction: given that we **don't use it ANYWHERE exept its own test**, my temptation is to conclude that we should remove it ...

Reply:
Correction to the correction: yes, we were using it in one place. Sorry about the noise.